menu/menu_shader.c

```diff
@@ -206,9 +206,9 @@ static const char *shader_get_preset_extension(unsigned type)
       case RARCH_SHADER_HLSL:
       case RARCH_SHADER_CG:
          return file_path_str(FILE_PATH_CGP_EXTENSION);
+      default:
+         return NULL;
    }
```

Comment:
Same here. Do a default: break instead, then return NULL at the end of the function.